File: php/09proc/proc56.php
<?php
function Leng($xA, $yA, $xB, $yB){
        return number_format(sqrt(pow($xA - $xB, 2) + pow($yA - $yB, 2)), 2);
}
$xA = mt_rand(1, 100);
$yA = mt_rand(1, 100);
$xB = mt_rand(1, 100);
$yB = mt_rand(1, 100);
$xC = mt_rand(1, 100);
$yC = mt_rand(1, 100);
$xD = mt_rand(1, 100);
$yD = mt_rand(1, 100);
echo "xA = ".$xA;
echo "<br>yA = ".$yA;
echo "<br>xB = ".$xB;
echo "<br>yB = ".$yB;
echo "<br>xC = ".$xC;
echo "<br>yC = ".$yC;
echo "<br>xD = ".$xD;
echo "<br>yD = ".$yD;
$AB = Leng($xA, $yA, $xB, $yB);
$AC = Leng($xA, $yA, $xC, $yC);
$AD = Leng($xA, $yA, $xD, $yD);
echo "<br>AB = ".$AB;
echo "<br>AC = ".$AC;
echo "<br>AD = ".$AD;
?>

File: cpp/09proc/Proc56.cpp
#include <iostream>
#include <cmath>
using namespace std;

struct Point
{
	double x;
	double y;
};

double Leng(double xA, double yA, double xB, double yB);

int main()
{
	//Task("Proc56");
	Point A, B;
	cin >> A.x >> A.y;
	for (int i = 1; i < 4; i++)
	{
		cin >> B.x >> B.y;
		cout << Leng(A.x, A.y, B.x, B.y);
	}
	
	return 0;
}

double Leng(double xA, double yA, double xB, double yB)
{
	return sqrt( pow(xB - xA, 2) + pow(yB - yA, 2) );
}